src/utils/shared/redirects.test.js

```diff
@@ -239,4 +239,11 @@ describe('getRedirects', () => {
       '/doc/use-cases/versioning-data-and-model-files'
     )
   })
+
+  describe('Does not accidentally redirect to an external site', () => {
+    itRedirects('https://dvc.org//google.com/', '/google.com')
+    itRedirects('//google.com/', '/google.com')
+    itRedirects('///evil.com/', '/evil.com')
+    itRedirects('/////////evil.com/', '/evil.com')
```

Comment:
2 of these tests are probably enough?
Also google = evil lol

Reply:
I removed the three slashes, but the remaining ones each cover enough to warrant inclusion IMO:
1. `//google.com/` - base case
2. `/////////evil.com/` - multiple slashes (to an extreme) which do trigger the bug when it happens
3. `http://dvc.org//google.com/` - including protocol